README.md

```diff
@@ -17,6 +17,7 @@ Tokens was initially developed for use in my private server, the idea was to giv
 -   **[CombatLogX](https://www.spigotmc.org/resources/combatlogx.31689/)**
 -   **[PlaceholderAPI](https://www.spigotmc.org/resources/placeholderapi.6245/)**
 -   **[Any Vault Economy](https://www.spigotmc.org/resources/vault.34315/)**
+-   **[Prison](https://www.spigotmc.org/resources/prison.1223/history/)**
```

Comment:
Is there support added for Prison?

Reply:
I'm sorry, I did not see your question.
Yes, prison supports the use of this plugin within it's BlockEvent commands.  Prior to these performance enhancements, use of this plugin within a BlockEvent was causing hundreds of hits on Tokens per minute, and in turn, was causing an immense amount of lag.  Now with these enhancements, it can support many players mining concurrently with thousands of hits to this plugin per minute without causing lag.  So yeah, Tokens was specifically enhanced to support use within prison.

Reply:
Alright, thanks for the response, just making sure!